### PR TITLE
Switched to +p(packagednumber)

### PR DESCRIPTION
**Pull request by djw-m**
## What Changed?

Draft of +p for clashing version numbers

### CODE REVIEW

**Review by mwanner — DISMISSED** [review]

File: product_docs/docs/pgd/4/rel_notes/pgd_4.3.1+p1_rel_notes.mdx [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
-title: "Release notes for EDB Postgres Distributed version 4.3.1-1"
-navTitle: "Version 4.3.1-1"
+title: "Release notes for EDB Postgres Distributed version 4.3.1+p1"
+navTitle: "Version 4.3.1+p1"
 ---
 
-EDB Postgres Distributed version 4.3.1-1 is a minor release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.
+EDB Postgres Distributed version 4.3.1+p1 is a minor release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.
```
Comment (LenzGr) [inline product_docs/docs/pgd/4/rel_notes/pgd_4.3.1+p1_rel_notes.mdx:6]: So is it "patch release" or "minor release"? What's the purpose of this `+p1`suffix? Do we explain this somewhere?

Comment (LenzGr) [reply product_docs/docs/pgd/4/rel_notes/pgd_4.3.1+p1_rel_notes.mdx:6]: Thanks for clarifying the meaning of the version suffix, @djw-m - for consistency, I'd like to propose we stick to calling these updates "patch releases" (abbreviated via the "p" suffix).

Comment (djw-m) [reply product_docs/docs/pgd/4/rel_notes/pgd_4.3.1+p1_rel_notes.mdx:6]: Semver says the third digit of the version number is the patch release number which is how we got here. @mwanner - want to express a preference on what we call these patch/rebundled releases?

Comment (mwanner) [reply product_docs/docs/pgd/4/rel_notes/pgd_4.3.1+p1_rel_notes.mdx:6]: I don't mind us calling this a patch release as well.

**Review by mwanner — APPROVED** [review]

**Pull request merged**